### DISCUSSION TITLE
Cannot access my Hubs anymore, Verify Email Unknown Error

### DISCUSSION THREAD

**Discussion by SamanthaZH** [mozilla/hubs#5595] [Hubs Cloud Questions]
After I add an allowed CROS url in admin setting, I cannot login  anymore. 
Login by command, it returns Unknown Error When I click the verifying link in inbox. 
And when I access browser, I cannot receive the verification email. 

<img width="537" alt="image" src="https://user-images.githubusercontent.com/67449285/179429415-25b45f7f-f626-4665-8776-6e5b829ceb40.png">

**Comment by Jsroyster** [comment]
I am having the same problem. I was able to access in July using jeremysroyster@gmail.com . Now I get the same error

**Comment by matthewbcool** [comment]
@Jsroyster  @SamanthaZH  Please see https://github.com/mozilla/hubs/issues/5720#issuecomment-1252804732 for a fix. Thanks!
